app/internal/user.py

```diff
@@ -0,0 +1,48 @@
+from app.database import models, schemas
+from sqlalchemy.orm import Session
+from app.internal.security.ouath2 import pwd_context
+
+# pwd_context = CryptContext(schemes=["bcrypt"])
```

Comment:
Remove if not necessary

Reply:
Please don't ever mark a comment as a resolved one unless you've addressed it.

I've asked it multiple times, both in the group and in the [forum](https://forums.pythonic.guru/t/code-review-peer-review/8862).